### ISSUE TITLE
Node.js client sends UDP packets that agent in all-in-one does not recognize

### ISSUE THREAD

**Issue by yurishkuro** [uber/jaeger#194]
I ran the [test app](https://gist.github.com/julianste/908612cdbac518eff5416fb8d89fafb6) reported on [StackOverflow](https://stackoverflow.com/questions/43766832/uber-jaeger-client-node-backend-wont-receive-data) and saw the agent logging the following error:
```
process error Expected protocol id 82 but got 80
```

The protocol id 82 is for compact protocol, but the Node.js client is expected to send binary protocol. Replacing log with panic:
```
panic: Expected protocol id 82 but got 80

goroutine 13 [running]:
github.com/uber/jaeger/vendor/github.com/apache/thrift/lib/go/thrift.(*TCompactProtocol).ReadMessageBegin(0xc4203e0000, 0x0, 0xc4201a96f8, 0x12c84e6, 0xc420084170, 0x0)
	/Users/yurishkuro/golang/src/github.com/uber/jaeger/vendor/github.com/apache/thrift/lib/go/thrift/compact_protocol.go:340 +0x4ab
github.com/uber/jaeger/thrift-gen/agent.(*AgentProcessor).Process(0xc42018c520, 0x1783540, 0xc4203e0000, 0x1783540, 0xc4203e0000, 0x0, 0x0, 0x0)
	/Users/yurishkuro/golang/src/github.com/uber/jaeger/thrift-gen/agent/agent.go:146 +0x41
github.com/uber/jaeger/cmd/agent/app/processors.(*ThriftProcessor).processBuffer(0xc420074c00)
	/Users/yurishkuro/golang/src/github.com/uber/jaeger/cmd/agent/app/processors/thrift_processor.go:116 +0x180
created by github.com/uber/jaeger/cmd/agent/app/processors.(*ThriftProcessor).Serve
	/Users/yurishkuro/golang/src/github.com/uber/jaeger/cmd/agent/app/processors/thrift_processor.go:88 +0x6e
exit status 2
```

Will continue troubleshooting tomorrow.

**Comment by vprithvi** [comment]
Doesn't the latest version of the node client send jaeger.thrift in the binary thrift protocol?
Additionally, the example configures the UDP to 5775. Shouldn't this be 6832?

**Comment by yurishkuro** [comment]
I made it working by removing the port config from example, as well as running the full docker command. The trace now gets into the storage, but the UI is having trouble displaying it - https://github.com/uber/jaeger-ui/issues/33

**Comment by yurishkuro** [comment]
UI issue fixed by #195
